ios/.ado/fabric-ios-publish-nuget.yml

```diff
@@ -0,0 +1,89 @@
+# This file defines the Apple publish build steps
```

Comment:
and we're good with the .ado folder living on GitHub? Or should that be entirely private since it's specific to ado?

Reply:
the CI for nuget package will live in ADO still right? although, there isn't a huge secret here so i don't mind ...

Reply:
I think that for some of our builds, specifically our internal NuGet publishing builds, we'll still want those to be ADO pipelines built from this source. This pattern exists similarly in https://github.com/microsoft/fluentui-react-native. I prefer to keep the build definitions under source control along with the source.